scripts/onboarding_script.js

```diff
@@ -309,7 +311,7 @@ const createCPUAction = (sponsor) => {
         authorization: [
             {
                 actor: harvest,
-                permission: 'active',
+                permission: 'payforcpu',
```

Comment:
That seems wrong - should be active.

Reply:
The `payforcpu` permission exists for the limited purpose of executing the `payforcpu` action (see key table below). While `active` is also capable of executing this, it is a more powerful authorization and IMHO it is good practice to use the least powerful auth that does the job.

In this particular case, I as a user do not hold the harvst.seeds active key so I could not execute the script. On the other hand the payforcpu key is not strictly controlled (I scraped it from the LW repo). In the end I think this level of auth makes more sense.
![image](https://user-images.githubusercontent.com/2141014/199758053-add5dff9-af52-4faf-8372-b962b8ec8a99.png)

Reply:
Oh - I see so that was a bug in the script.